tests/framework/types.go

```diff
@@ -39,22 +40,22 @@ type VirtualMachine struct {
 
 func (vm VirtualMachine) Create() (string, string, error) {
 	args := []string{"create", "-f", vm.Manifest}
-	return ktests.RunCommandWithNS(vm.Namespace, ktests.KubeVirtOcPath, args...)
+	return ktests.RunCommandWithNS(vm.Namespace, ocName, args...)
```

Comment:
`ktests.KubeVirtOcPath` is configurable from command line, I don't find this change as an improvement.

Reply:
@lukas-bednar I've rewrote it using hack/run-tests.sh (as you suggested in the email) and shell ENV